### PR TITLE
Add documentation for pilight dimmer as light

### PR DESCRIPTION
**Pull request by Trekky12**
**Description:**
Add configuration parameters for pilight dimmer as light component

**Pull request in home-assistant:** home-assistant/home-assistant#30107

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/pilight.markdown [hunk 1]
```diff
@@ -22,6 +22,7 @@ There is currently support for the following device types within Home Assistant:
 - [Binary Sensor](#binary-sensor)
 - [Sensor](#sensor)
 - [Switch](#switch)
+- [Dimmer](#dimmer)
```
Comment (frenck) [inline source/_integrations/pilight.markdown]: We don't have a Dimmer platform, I guess you meant to use "Light" here?

File: source/_integrations/pilight.markdown [hunk 2]
```diff
@@ -304,6 +305,53 @@ switch:
...
+          id: 23298822
+          unit: 10
+          state: 'on'
+        off_code_receive:
+          protocol: kaku_dimmer
+          id: 23298822
+          unit: 10
+          state: 'off'
+```
+
```
Comment (frenck) [inline source/_integrations/pilight.markdown:353]: Remove additional empty line
```suggestion
```

File: source/_integrations/pilight.markdown [hunk 3]
```diff
@@ -304,6 +305,53 @@ switch:
           state: 'off'
 ```
 
+## Dimmer
+
+Pilight dimmer devices, which can have different brightness values can be used as light component.
```
Comment (frenck) [inline source/_integrations/pilight.markdown]:
```suggestion
Pilight dimmer devices, which can have different brightness values, can be used as a light. 
```

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for the PR @Trekky12!

I've reviewed it, and left you some comments. Could you please take a look? Thanks! 👍

**Comment by Trekky12** [comment]
Thank you for your feedback, I changed it accordingly :)

**Review by frenck — APPROVED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Pull request merged**